frontend/app/components/import_list/import.component.html

```diff
@@ -5,12 +5,21 @@ <h5 class="card-title mb-0">Liste des imports</h5>
         </div>
         <div class="card-body">
             <div *ngIf="!empty">
-                <ngx-datatable #table class="material striped" [rows]="history" [columnMode]="'force'"
+                <input
+                    #search
```

Comment:
```suggestion
                    [formControl]="search"
```
On binde le formControl sur l'input